src/configLoader/getContent.js

```diff
@@ -61,3 +61,25 @@ function getConfigContent (configPath, baseDirectory) {
     const content = readConfigContent(resolvedPath);
     return getNormalizedConfig(configBasename, content);
 };
+
+/**
+ * Read proper content from config file.
+ * If the chartset of the config file is not utf-8, one error will be thrown.
+ * @param {String} configPath 
+ * @return {String}
+ */
+function readConfigFileContent (configPath) {
+
+    if (!configPath) {
```

Comment:
Can we change this to `if (typeof configPath !== 'string')` and throw a `TypeError` instead?